install-core.yaml.tmpl

```diff
@@ -103,13 +103,13 @@ spec:
           imagePullPolicy: Always
           env:
             - name: CORE_INSTANCE_TAGS
-              value: 
+              value: $CORE_INSTANCE_TAGS
```

Comment:
My concern is that this will require us to ensure the Core UI is updated as well and anyone using it now provides that variables - otherwise they will end up with `null` values.

Reply:
Ah no `null` is empty string.

Reply:
Still needs people to run the updated `envsubst` command though otherwise they'll end up with tags of $CORE_INSTANCE_TAGS and name of $CORE_INSTANCE_NAME.